community/gsoc_responsibilities.md

```diff
@@ -2,7 +2,7 @@ Responsibilities of coala’s GSoC Participants
 =============================================
 
 This Document hold the responsibilities we expect coala’s GSoC participants to
-fulfil.
+fulfil including coala community's [Code of Conduct](https://coala.io/coc).
```

Comment:
This should be a new sentence, indicating that the CoC document is a responsibility for all, mentors and students alike.
To finish this task you need to improvise the wording here based on your own understanding of the CoC document.